### PR TITLE
Support cron deploys

### PR DESCRIPTION
**Pull request by meatballhat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,12 +12,22 @@ pull docker containers.
 
 ``` yaml
 ---
+# Application name used in file paths, service actions, etc.
+modcloth_app_name: app
+
+# The type of application to be deployed, either "long-running" or "cron"
+modcloth_app_type: long-running
```
Comment (jszwedko) [inline README.md:19]: Would it make sense to use two separate roles (that perhaps depend on the same role for identical tasks) rather than conditionals?

**Comment by jszwedko** [comment]
:shipit: if we decide to go the single role route.

File: README.md (hunk 1 above)
Comment (meatballhat) [inline README.md:19]: Or we could do a conditional include trick of separate task files, maybe?

Comment (meatballhat) [inline README.md:19]: I'm frobnicating.  1 sec.

**Comment by jszwedko** [comment]
:+1: this feels better

**Comment by alexschlessinger** [comment]
looks good to me

File: tasks/main.yml [hunk 2]
```diff
@@ -22,11 +14,4 @@
...
 
-- name: stop app
-  service: name={{ modcloth_app_name }} state=stopped
-  when: modcloth_app_write_upstart_conf.changed or
-        modcloth_app_write_etc_default_file.changed or
-        modcloth_app_pull_container.changed
-
-- name: start app
-  service: name={{ modcloth_app_name }} state=started
+- include: "{{ modcloth_app_type }}.yml"
```
Comment (rafecolton) [inline tasks/main.yml:17]: I thought an include with a var in the name didn't work?

File: templates/cron-wrapper.sh.j2 [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# {{ ansible_managed }}
+set -e
+
+DOCKER_SOCK=/var/run/docker.sock
+
+while [ ! -e $DOCKER_SOCK ] ; do
+  inotifywait -t 2 -e create $(dirname $DOCKER_SOCK)
```
Comment (rafecolton) [inline templates/cron-wrapper.sh.j2]: Thought we were getting rid of this stuff?

**Comment by rafecolton** [comment]
LGTM pending a few small questions

File: tasks/main.yml (hunk 2 above)
Comment (meatballhat) [inline tasks/main.yml:17]: nope works fine `¯\_(ツ)_/¯`

File: templates/cron-wrapper.sh.j2 (hunk 3 above)
Comment (meatballhat) [inline templates/cron-wrapper.sh.j2]: yup sure are

File: tasks/main.yml (hunk 2 above)
Comment (rafecolton) [inline tasks/main.yml:17]: :thumbsup:

File: templates/cron-wrapper.sh.j2 (hunk 3 above)
Comment (rafecolton) [inline templates/cron-wrapper.sh.j2]: Ok, LMK if it's good to merge

:shipit:

**Comment by meatballhat** [comment]
If/when this gets merged, I'd like to tag `v0.2.0`.  Sound OK?

**Comment by rafecolton** [comment]
Works for me

**Pull request merged**